### DISCUSSION TITLE
Very slow on large Git repos

### DISCUSSION THREAD

**Discussion by marekful** [ohmyzsh/ohmyzsh#9801] [Help] [answered]
Hi Community,

I've been using and loving Oh My zsh for quite some time now. On a brand new MacBookPro16,1 (Bug Sur) when I cd into a directory holding a large Git repository, I get really slow response when hitting Enter. E.g. when I press Enter 3 times in quick succession, it takes ~ 4 seconds (!) until I get the prompt back. This is the same with Terminal and iTerm2.

Is there any way to fix/mitigate this?

Thanks,
Marcell

**Comment by signed-log** [comment, marked as answer]
Yes, there is

Uncomment that variable in `~/.zshrc` and set that variable to true

```bash
# Uncomment the following line if you want to disable marking untracked files
# under VCS as dirty. This makes repository status check for large repositories
# much, much faster.
DISABLE_UNTRACKED_FILES_DIRTY="true"
```

**Reply by marekful** [reply]
Awesome, thanks!
